src/DynamoCoreWpf/ViewModels/Menu/PreferencesViewModel.cs

```diff
@@ -553,7 +545,6 @@ public bool IsolateSelectedGeometry
             }
             set
             {
-                isolateSelectedGeometry = value;
```

Comment:
why are we removing these setters here? Are these not used anymore?

Reply:
@zeusongit Only removing the private value setter since these private values are used. Suggested by VS